process/src/MainProcessor.ts

```diff
@@ -0,0 +1,165 @@
+      gameData.checkoutVersion(version);
+      console.timeEnd(`Checking out v${version.id} took`);
+    } else {
+      gameData.checkoutMaster();
+    }
+
+    console.log("\nPreparing directories...");
+    console.time("Preparing directories took");
+    gameData.prepareStaticDir();
+    console.timeEnd("Preparing directories took");
+
+    console.log("\nImporting objects...");
+    console.time("Importing objects took");
+    gameData.importObjects();
+    console.timeEnd("Importing objects took");
+
+    console.log("\nImporting categories...");
+    console.time("Importing categories took");
+    gameData.importCategories();
+    console.timeEnd("Importing categories took");
+
+    console.log("\nImporting transitions...");
+    console.time("Importing transitions took");
+    gameData.importTransitions();
+    console.timeEnd("Importing transitions took");
+
+    console.log("\nImporting biomes...");
+    console.time("Importing biomes took");
+    gameData.importBiomes();
+    console.timeEnd("Importing biomes took");
+
+    console.log("\nPopulating versions...");
+    console.time("Populating versions took");
+    gameData.populateVersions();
+    console.timeEnd("Populating versions took");
+
+    console.log("\nCalculating object depth...");
+    console.time("Calculating object depth took");
+    gameData.calculateObjectDepth();
+    console.timeEnd("Calculating object depth took");
+
+    if (this.doSprites) {
+      console.log("\nConverting sprite images...");
+      console.time("Converting sprite images took");
+      gameData.convertSpriteImages();
+      console.timeEnd("Converting sprite images took");
+
+      console.log("\nConverting ground images...");
+      console.time("Converting ground images took");
+      gameData.convertGroundImages();
+      console.timeEnd("Converting ground images took");
+
+      console.log("\nProcessing sprites...");
+      console.time("Processing sprites took");
+      gameData.processSprites();
+      console.timeEnd("Processing sprites took");
+    }
+
+    if (this.doSounds) {
+      console.log("\nConverting sound files...");
+      console.time("Converting sound files took");
+      gameData.convertSounds();
+      console.timeEnd("Converting sound files took");
+    }
+
+    console.log("\nExporting objects...");
+    console.time("Exporting objects took");
+    gameData.exportObjects();
+    console.timeEnd("Exporting objects took");
+
+    // console.log("\nExporting versions...");
+    // gameData.exportVersions();
+
+    console.log("\nExporting biomes...");
+    console.time("Exporting biomes took");
+    gameData.exportBiomes();
+    console.timeEnd("Exporting biomes took");
+
+    if (version) {
+      console.log("\nGenerating sitemap...");
+      console.time("Generating sitemap took");
+      gameData.generateSitemap();
+      console.timeEnd("Generating sitemap took");
+      return null;
+    }
+
+    console.timeEnd("Processing took");
```

Comment:
Need to copy line 155 to under line 151, to catch the timeEnd for when we do process a versioned data set.

Reply:
Whoops, missed that! That'll be an easy one to tackle in the new issue.